addon/components/paper-button.js

```diff
@@ -4,45 +4,53 @@ import RippleMixin from '../mixins/ripple-mixin';
 import ProxiableMixin from 'ember-paper/mixins/proxiable-mixin';
 import ColorMixin from 'ember-paper/mixins/color-mixin';
 
+const {
+  assert,
+  computed,
+  isPresent
+} = Ember;
+
 export default BaseFocusable.extend(RippleMixin, ProxiableMixin, ColorMixin, {
   attributeBindings: ['target', 'action', 'type'],
   tagName: 'button',
   themed: true,
-  classNameBindings: ['raised:md-raised', 'icon-button:md-icon-button', 'focus:md-focused', 'themed:md-default-theme', 'themed:md-button'],
+  classNameBindings: [
+    'raised:md-raised',
+    'icon-button:md-icon-button',
+    'focus:md-focused',
+    'themed:md-default-theme',
+    'themed:md-button'
+  ],
 
   // Ripple Overrides
   rippleContainerSelector: null,
-  fitRipple: Ember.computed.readOnly('isIconButton'),
-  center: Ember.computed.readOnly('isIconButton'),
-  dimBackground: Ember.computed.not('isIconButton'),
+  fitRipple: computed.readOnly('isIconButton'),
+  center: computed.readOnly('isIconButton'),
+  dimBackground: computed.not('isIconButton'),
 
-  noSpan: Ember.computed('no-span', function() {
-    return this.get('no-span');
-  }),
+  noSpan: computed.readOnly('no-span'),
 
   // RippleMixin overrides
-  isIconButton: Ember.computed(function() {
-    return this.classNames.any(function(className) {
-      return className.indexOf('md-icon-button') !== -1;
-    });
+  isIconButton: computed({
+    get() {
```

Comment:
I honestly just left this property there in case it was used by some parent class, but seems pointless from what I've seen. Should I remove it?